samples/eShopLite/nodeapp/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
```

Comment:
Make a new sample. Don't force everyone to download node 😄

Reply:
This isn't staying here -- no one will need to download node ;)